src/validation.ts

```diff
@@ -19,6 +19,24 @@ const JsonSchema: z.ZodType<Json> = z.lazy(() =>
 
 const PublicKeySchema = z.string().length(55).startsWith('B62');
 
+interface ProofType {
+  name: string;
+  publicInput: SerializedType;
+  publicOutput: SerializedType;
+  maxProofsVerified: number;
+  featureFlags: Record<string, unknown>;
+}
+
+interface SerializedType {
+  _type?: string;
+  type?: 'Constant';
```

Comment:
is that the correct type? it's unintentional that we still use `type` for constants 😅 can you maybe fix that?